_data/companies/px.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: PX
```

Comment:
What does PX stand for?

Reply:
PX = Private Exchange, PX platform, PX inc.

Reply:
```suggestion
name: PX (Private Exchange)
```

What do you think about this change?

Reply:
I would prefer 'PX Platform'